frontend/src/Editor/Components/PasswordInput.jsx

```diff
@@ -18,6 +18,16 @@ export const PasswordInput = ({
   const [passwordValue, setPasswordValue] = React.useState('');
   const { isValid, validationError } = validate(passwordValue);
 
+  const computedStyles = {
+    height,
+    display: visibility ? '' : 'none',
+    borderRadius: `${borderRadius}px`,
+    color: darkMode && '#fff',
+    borderColor: darkMode && '#DADCDE',
+    backgroundColor: darkMode && '#232e3c',
```

Comment:
When background color is changed from styles tab in widget inspector it wont be applied here , you can take reference from `TextInput.jsx` to resolve this